llama-index-experimental/llama_index/experimental/embedding_finetuning/base.py

```diff
@@ -0,0 +1,172 @@
+import logging
+from typing import Any, Optional
+
+import torch
```

Comment:
I don't think `torch` is in the dependencies for experimental

Best to either hide this import under a try/except where its needed, and print out a proper error message with the pip install